ImplementationGuide/markdown/ISiKundSMART/Schritt6RefreshToken.md

```diff
@@ -1,23 +1,19 @@
 # Schritt 6: Refresh Token & Revocation
 
----
-### Informativ
----
-
 ## Refresh Tokens
 
 Die Gültigkeitsdauer eines Access-Tokens kann beliebig durch das bestätigungsrelevante System gewählt werden. Mit Hinblick auf [RFC6819 - OAuth 2.0 Threat Model and Security Considerations - Limited Access Token Lifetime](https://datatracker.ietf.org/doc/html/rfc6819#section-3.1.2) soll die Gültigkeitsdauer auf Minuten oder Stunden beschränkt werden. Das Gültigkeitsdatum wird im Access Token durch den Parameter "expires_in" angegeben. Alternativ kann die Gültigkeit des Tokens über den Introspection-Endpunkt abgefragt werden.
 
 Um eine häufige Authentifizierung der Benutzer und/oder des Clients zu vermeiden, soll der Autorisierungsserver die Ausstellung von Refresh Tokens unterstützen.
 
-Folgende Anforderungen aus [SMART App Launch - 2.0.12 - Refresh access token](https://hl7.org/fhir/smart-app-launch/STU2/app-launch.html#refresh-access-token) sind bei der Unterstützung eines Token Refresh zu beachten:
+Folgende Anforderungen aus [SMART App Launch - 2.1.12 - Refresh access token](https://hl7.org/fhir/smart-app-launch/STU2.1/app-launch.html#refresh-access-token) sind bei der Unterstützung eines Token Refresh zu beachten:
 
 - Der "offline_access" Scope muss unterstützt werden, der "online_access" Scope soll unterstützt werden.
 - Ein Refresh Token muss an die anfordernde Client Id gebunden werden, d.h. eine abweichende Client Id (übermittelt durch die Autorisierung des Clients) DARF NICHT verwendbar sein, um ein neues Access Token auszustellen.
 - Der Autorisierungsserver muss bei der Ausstellung eines neuen Access Tokens sicherstellen, dass im Vergleich zum originalen Access Token nur ein Subset oder die gleiche Menge an Scopes erlaubt werden. Fordert der Client neue Scopes an, soll das System die Menge aller Scopes auf die zuvor erlaubten Scopes einschränken.
 - Der Launch Context muss zurückgegeben werden, falls die entsprechenden Scopes durch den Client oder im originalen Access Token angefordert wurden.
 
-Alle verpflichtenden Implementierungsdetails aus [SMART App Launch - 2.0.12 - Refresh access token](https://hl7.org/fhir/smart-app-launch/STU2/app-launch.html#refresh-access-token) müssen bei der Umsetzung eines Token Refresh durch den Autorisierungsserver unterstützt werden.
+Alle verpflichtenden Implementierungsdetails aus [SMART App Launch - 2.1.12 - Refresh access token](https://hl7.org/fhir/smart-app-launch/STU2.1/app-launch.html#refresh-access-token) müssen bei der Umsetzung eines Token Refresh durch den Autorisierungsserver unterstützt werden.
 
 ----
 
```

Comment:
issue: in der nächsten Zeile wird ein Beispiel eingeleitet, das Beispiel an sich fehlt aber